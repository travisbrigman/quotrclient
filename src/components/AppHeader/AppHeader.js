import { Box } from "grommet";

export default function AppHeader(props) {
  return (
    <Box
      tag="header"
      direction="row"
      align="center"
      justify="between"
      background="brand"
      pad={{ left: "medium", right: "small", vertical: "small" }}
      elevation="medium"
      style={{ zIndex: "1" }}
      {...props}
    />
  );
}
