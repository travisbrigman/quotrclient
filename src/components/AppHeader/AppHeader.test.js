import AppHeader from "./AppHeader";
import { render } from "@testing-library/react";

test("can render", () => {
  const { container } = render(
    <AppHeader>
      <p>Header</p>
    </AppHeader>
  );

  expect(container).toMatchSnapshot();
});
