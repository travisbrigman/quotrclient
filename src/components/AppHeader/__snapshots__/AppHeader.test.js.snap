// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render 1`] = `
<div>
  <header
    class="StyledBox-sc-13pk1d4-0 fCjba"
    style="z-index: 1;"
  >
    <p>
      Header
    </p>
  </header>
</div>
`;
